openhtf/plugs/uart_device.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2014 Google Inc. All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""SerialPlug used for serial communication
+Requires the following keys in .yaml configuration file AND declared.
+
+.yaml file
```

Comment:
Strongly agree.  In fact I tried this earlier but couldn't get it to work in the hierarchical format.  Got it working now. 

Will do the following:
plug name1:
  config key1

plug name2:
 config key1
 config key2